docs/examples.rst

```diff
@@ -278,30 +278,30 @@ When unzipped, the model bundle contains a ``model.pth`` file which can be used
    * - SpaceNet Rio Buildings
      - Chip Classification
      - Resnet 50
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-rio-cc/model-bundle.zip>`_
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-rio-cc/013022223130_sample.tif>`_
+     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.13/spacenet-rio-cc/model-bundle.zip>`_
+     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.13/spacenet-rio-cc/sample-predictions/sample-img-spacenet-rio-cc.tif>`_
    * - SpaceNet Vegas Buildings
      - Semantic Segmentation
      - DeeplabV3 / Resnet50
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-vegas-buildings-ss/model-bundle.zip>`_
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-vegas-buildings-ss/1929.tif>`_
+     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.13/spacenet-vegas-buildings-ss/model-bundle.zip>`_
+     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.13/spacenet-vegas-buildings-ss/sample-predictions/sample-img-spacenet-vegas-buildings-ss.tif>`_
    * - SpaceNet Vegas Roads
      - Semantic Segmentation
      - DeeplabV3 / Resnet50
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-vegas-roads-ss/model-bundle.zip>`_
-     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.12/spacenet-vegas-roads-ss/524.tif>`_
+     - `link <https://s3.amazonaws.com/azavea-research-public-data/raster-vision/examples/model-zoo-0.13/spacenet-vegas-roads-ss/model-bundle.zip>`_
```

Comment:
Aren't you using EfficientNet and not DeepLabV3 now that the `external_model` field is set to True?

Reply:
That is only for isprs_potsdam and it uses an FPN. But yes, it should be corrected here.